geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/clustering/CentroidManagerGeoWave.java

```diff
@@ -283,12 +291,23 @@ public CentroidManagerGeoWave(
 				CentroidParameters.Centroid.ZOOM_LEVEL,
 				1);
 
-		basicAccumuloOperations = new BasicAccumuloOperations(
-				runTimeProperties.getProperty(GlobalParameters.Global.ZOOKEEKER),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_INSTANCE),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_USER),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_PASSWORD),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_NAMESPACE));
+		try {
+			basicAccumuloOperations = runTimeProperties.getInstance(
+					CommonParameters.Common.ACCUMULO_CONNECT_FACTORY,
+					this.getClass(),
+					BasicAccumuloOperationsFactory.class,
+					DirectBasicAccumuloOperationsFactory.class).build(
+					runTimeProperties.getProperty(GlobalParameters.Global.ZOOKEEKER),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_INSTANCE),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_USER),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_PASSWORD),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_NAMESPACE));
+		}
+		catch (InstantiationException | IllegalAccessException e) {
```

Comment:
Gotcha - I understand it in the case of a constrained method that needs to be implemented/overrided - just didn't see that here.   What does e.getMessage() do in this case?   Is there any value in something like  
```Java
throw new IOException(e, e.getMessage())
```
(might be pointless - intent here is just to make sure we aren't slightly obfuscating the InstatiantionException of IllegalAcessException, since those are more useful than IOException)